app/src/main/java/org/gophillygo/app/data/DestinationDao.java

```diff
@@ -100,7 +100,6 @@ public void bulkUpdate(List<Destination> destinations) {
      *
      * @return One randomized image for each of the home page grid of filter categories
      */
-    @Transaction
```

Comment:
Out of curiousity, why remove `@Transaction` from this method?

Reply:
That change came from applying a suggested change from the linter. I think it had to do with something about how it couldn't be in a transaction, or already was in a transaction.